libraries/AP_Camera/AP_Camera.h

```diff
@@ -47,11 +47,21 @@ class AP_Camera {
     // enums
     enum class CameraType {
         NONE = 0,           // None
+#if AP_CAMERA_SERVO_ENABLED
```

Comment:
I don't think there's any reason to get rid of the enums..

Reply:
I *really* like getting rid of the enums!  When combined with switch() statements it really helps ensure you don't have stray code floating around which should be being excluded but isn't.  I've based a few PRs on this fact in the past.